src/C-interface/ellblock/bml_element_multiply_ellblock_typed.c

```diff
@@ -0,0 +1,332 @@
+#include "../../internal-blas/bml_gemm.h"
```

Comment:
@ares201005 It looks like this function is never called by the higher level interface functions, right? Is it because it is not working? I'd prefer a PR without this file then. If it is working, then the test for it should be activated

Reply:
right, I need to remove it in this PR. This are still some issues to be fixed. The ellblock and distributed2d will be in the next PR.